ChatQnA/docker/xeon/README.md

```diff
@@ -57,6 +57,34 @@ chaqna-xeon-ui-server
 Port 5173 - Open to 0.0.0.0/0
 ```
 
+You can pull ChatQnA and components docker images form docker hub or build images from source code.
+
+## 🚀 Pull docker images
```

Comment:
No need to pull the image manually, when you run docker compose, the image will be pull automatically.

Reply:
Exactly, there is no need to pull the Docker images explicitly at all.

Reply:
Hold on this change as some images are behind in terms of approval